api/k8sdeps/kunstruct/factory.go

```diff
@@ -76,6 +99,13 @@ func (kf *KunstructuredFactoryImpl) FromMap(
 // MakeConfigMap returns an instance of Kunstructured for ConfigMap
 func (kf *KunstructuredFactoryImpl) MakeConfigMap(
 	kvLdr ifc.KvLoader, args *types.ConfigMapArgs) (ifc.Kunstructured, error) {
+	if kf.addAnnotation {
+		anno := args.Options.Annotations
+		if anno == nil {
+			anno = make(map[string]string)
+		}
+		anno[builtFromKustomizeAnnotation] = kustomizeVersion
+	}
```

Comment:
it appears anno is dropped on the floor

can we reuse `updateAnnotation` here ?

Reply:
oh nevermind, its a reference.